docs/reference/snaps.rst

```diff
@@ -0,0 +1,76 @@
+.. _snap_reference:
+
+===============
+Snap Versions
+===============
+
+There are several checkbox snaps and channels to choose from. You can
+get a full list via ``snap info checkbox``. You can refer to the following
+to decide what you should install.
+
+.. _ref_which_snap:
+
+Picking your version
+====================
+
+As you may recall from the installation guide, when installing checkbox
```

Comment:
You can add a link to the installation tutorial in case the reader got on this page from somewhere other than the installation tutorial :)